app-office/planner/planner-0.14.6_p20130520-r3.ebuild

```diff
@@ -63,4 +65,5 @@ src_install() {
 	if ! use examples; then
 		rm -rf "${D}/usr/share/doc/${PF}/examples"
 	fi
+	mv "${ED}"/usr/share/doc/planner "${ED}"/usr/share/doc/${PF} || die
```

Comment:
This addition points out a flaw in the previous version, which already tried to address this. Namely everything above your addition already is about installing these in the right place, but that doesn't work for some reason anymore and thus the docs and examples still get installed in /usr/share/doc/planner/ (that the QA complains about) and now you just move them over - but examples are still installed regardless of the `rm -rf` above, as they aren't in `${PF}` to remove, etc.

The above should get fixed, or replaced with your version in a way where USE=examples is honored

Reply:
I know this is not as delicate as it used to be know, it's rather sledgehammer, but it works and it doesn't break as easily if upstream decides to do things different.